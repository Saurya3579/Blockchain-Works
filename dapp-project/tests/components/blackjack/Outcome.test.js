import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

import Outcome from '../../../src/components/blackjack/Outcome';

test('should render NoMatch component', () => {
    const wrapper = shallow(<Outcome />);
    expect(toJSON(wrapper)).toMatchSnapshot();
});