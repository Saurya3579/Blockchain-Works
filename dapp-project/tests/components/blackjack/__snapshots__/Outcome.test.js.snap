exports[`test should render NoMatch component 1`] = `
<div
  className="alert alert-info"
  role="alert">
  Hit or Stand
</div>
`;
